server/helpers/UserHelper.js

```diff
@@ -0,0 +1,35 @@
+import { User } from '../sequelize/models/index';
+
+/**
+ * This class contains all methods
+ * required to handle
+ * user data to the database.
+ */
+class userhelper {
+  /**
+   * This method updates the user in the database.
+   * @param {Number} id The id of user to update
+   * @param {object} data The data to create a user.
```

Comment:
Change to **User information to be updated**

Reply:
done